integrations/adp/nango.yaml

```
@@ -0,0 +1,74 @@
+        title: string
+        department:
+        id: string
+        name: string
+        employmentType: FULL_TIME | PART_TIME | CONTRACTOR | INTERN | TEMPORARY | OTHER
+        employmentStatus: ACTIVE | TERMINATED | ON_LEAVE | SUSPENDED | PENDING
+        startDate: string
+        terminationDate?: string
+        terminationType?: string
+        manager?: Person
+        workLocation:
+        name: string
```

YAMLlint (1.37.1)

[error] duplication of key "name" in mapping

(key-duplicates)